.pre-commit-config.yaml

```diff
@@ -7,10 +7,6 @@ repos:
     rev: v0.7.1.1
     hooks:
       - id: shellcheck
-  - repo: https://github.com/prettier/prettier
```

Comment:
Removing it from now since it is failing.

Reply:
Is it just this version of prettier?

Reply:
I was being a bit lazy. I did not want to investigate what it would take to fix it.
It seems they changed the repository and they use a different tagging system.

Reply:
ok, no worries. Someone can add it back in if they have the time to look into it.

Reply:
FYI, at this point, this is an outdated conversation since I fixed the PR to switch to the new prettier pre-commit repo.